### DISCUSSION TITLE
Disable Versioning

### DISCUSSION THREAD

**Discussion by SelfhostedPro** [sqlalchemy/alembic#811] [Usage Questions]
**Is your feature request related to a problem? Please describe.**
I've got a projects that's built with docker and due to the nature of containers I don't want the versions to be persistent as the versions will be deleted even if the database is not.

**Describe the solution you'd like**
I would like to be able to disable versioning (the main thing is the db entry) so I don't have to drop the table every boot. A flag in the ini would be much appreciated.

**Describe alternatives you've considered**
I could just drop it every time but that requires me to add sqlite to my image which increases size.

**Additional context**
The project is [Yacht](https://github.com/selfhostedpro/yacht)

**Have a nice day!**

**Comment by zzzeek** [comment]
hey there -

I'm not following what it is you are trying to do.  if you don't have an alembic_version table then what are you using Alembic for ?

**Reply by SelfhostedPro** [reply]
I'm using it to add tables to the database when new ones are added (runs on container start to make sure all of the tables are added) but users don't need the ability to roll them back so I don't need the alembic_version database table. 

Currently I'm manually dropping the versions table every boot (if it exists) because the versions folder isn't persistent and will throw an error if it's unable to find the version in that folder.

**Reply by zzzeek** [reply]
the alembic version table is necessary so it knows what version to start with.  if you are starting from zero and going to the end on every run, you don't need alembic.  just call metadata.create_all().

**Reply by zzzeek** [reply]
otherwise, add a drop_table command to your env.py script at the end.   "connection.execute(text("DROP TABLE alembic_version"))"

**Reply by SelfhostedPro** [reply]
Ah, I'll add it to the env.py so I don't have to include sqlite with my container version. Know it's not a standard usecase for this tool. I appreciate the help!
